src/services/offerService.js

```diff
@@ -4,7 +4,7 @@ import Offer from "../components/HomePage/SearchResultsArea/Offer/Offer";
 import config from "../config";
 const { API_HOSTNAME } = config;
 
-const parseSearchFiltersToURL = (filters) => Object.keys(filters)
+export const parseSearchFiltersToURL = (filters) => Object.keys(filters)
```

Comment:
I suggest moving this function to the `/utils` folder and renaming it to `parseFiltersToURL` since it can be reused in other places.